rayon-core/src/thread_pool/mod.rs

```diff
@@ -80,6 +80,41 @@ impl ThreadPool {
     /// thread-local data from the current thread will not be
     /// accessible.
     ///
+    /// # Warning: execution order
+    ///
+    /// `install()` implicitly calls [`ThreadPool::yield_now()`],
+    /// potentially scheduling another task to run on the current thread. For
+    /// example:
```

Comment:
I would like this to clarify the context a bit more, something like:
```suggestion
    /// If the current thread is part of a different thread pool, it will try to
    /// keep busy while the `op` completes in its target pool, similar to
    /// calling [`ThreadPool::yield_now()`] in a loop. Therefore, it may
    /// potentially schedule other tasks to run on the current thread in the
    /// meantime. For example:
```

Does that make sense to you?

Reply:
Yes, changed to your phrasing via force push.